utils/src/main/kotlin/Constants.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.utils
+
+const val ORT_NAME = "ort"
```

Comment:
How about encapsulating them into an `object` for discoverability and for not polluting the global namespace too much? ..like e.g.
```
OrtConstants : object {
  companion object {
     NAME
     FULL_NAME
     DATA_DIR_ENV_NAME
     ...
  }
}
```
This would probably also make the imports in the code using these constants a bit nicer.

Reply:
I gave it a try, following the example of `SpdxConstants`, but as e.g. `ORT_NAME` is used in quite a lot of places, I like it being as short as possible instead of `OrtConstants.NAME`. At least for now.

Reply:
Consider `OrtConstants.ORT_NAME` then. I conjunction with a static import of the variable the code using it would still just write `ORT_NAME` but it gives discoverability. The imports are not beautified anymore then however.

Reply:
Right, I believe getting fewer imports if multiple constants are used was the main goal. I'll keep that idea in mind for a possible future PR.